rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/schedule/Sw360ScheduleService.java

```diff
@@ -42,4 +43,131 @@ public RequestStatus cancelAllServices(User sw360User) throws TException {
         }
     }
 
+    public RequestSummary svmSync(User sw360User) throws TException {
+        String serviceName = ThriftClients.SVMSYNC_SERVICE;
+        try {
+            if (PermissionUtils.isUserAtLeast(UserGroup.ADMIN, sw360User)) {
+                RequestSummary requestSummary = new ThriftClients().makeScheduleClient().scheduleService(serviceName);
+                return requestSummary;
+            } else {
+                throw new RuntimeException("User is not admin");
+            }
+        } catch (TException e) {
+            throw new TException(e.getMessage());
+        }
```

Comment:
This exception handling makes no sense as `TException` is captured and thrown.

I would rather recommend logging the exception with `log.error("Some appropriate message here", e)` and re-throw the captured exception `e`.